src/plugins/workspace/public/components/workspace_list/index.tsx

```diff
@@ -361,125 +436,152 @@ export const WorkspaceList = ({ registeredUseCases$ }: WorkspaceListProps) => {
-        },
-        {
-          name: 'Delete',
-          type: 'button',
-          description: 'Delete workspace',
-          'data-test-subj': 'workspace-list-delete-icon',
-          available: () => isDashboardAdmin,
-          render: (item: WorkspaceAttribute) => {
-            return (
-              <EuiButtonEmpty
-                onClick={() => {
-                  setDeletedWorkspaces([item]);
-                }}
-                size="s"
-                iconType="trash"
-                color="danger"
-                style={{ padding: 0 }}
-              >
-                Delete
-              </EuiButtonEmpty>
-            );
-          },
-        },
-      ],
+      actions: availableActions,
     },
   ];
 
-  return (
-    <EuiPage paddingSize="m">
-      <HeaderControl
-        controls={[
-          {
-            description: i18n.translate('workspace.list.description', {
-              defaultMessage: 'Organize collaborative projects with use-case-specific workspaces.',
-            }),
-          },
-        ]}
-        setMountPoint={application?.setAppDescriptionControls}
-      />
-      {isDashboardAdmin && renderCreateWorkspaceButton()}
-      <EuiPageContent
-        verticalPosition="center"
-        horizontalPosition="center"
-        panelPaddingSize="l"
-        paddingSize="m"
-        hasShadow={false}
-      >
-        <EuiInMemoryTable
-          compressed={true}
-          items={newWorkspaceList}
-          columns={columns}
-          itemId="id"
-          message={emptyStateMessage}
-          onTableChange={({ page: { index, size } }) =>
-            setPagination((prev) => {
-              return { ...prev, pageIndex: index, pageSize: size };
-            })
-          }
-          pagination={pagination}
-          sorting={{
-            sort: {
-              field: initialSortField,
-              direction: initialSortDirection,
+  const columns = [...availableColumns, ...actionColumns];
+
+  const workspaceListTable = (
+    <EuiInMemoryTable
+      compressed={true}
+      items={newWorkspaceList}
+      columns={columns}
+      itemId="id"
+      message={emptyStateMessage}
+      onTableChange={({ page: { index, size } }) =>
+        setPagination((prev) => {
+          return { ...prev, pageIndex: index, pageSize: size };
+        })
+      }
+      pagination={pagination}
+      sorting={{
+        sort: {
+          field: initialSortField,
+          direction: initialSortDirection,
+        },
+      }}
+      isSelectable={selectable}
+      search={searchable ? search : undefined}
+      selection={selectable ? selectionValue : undefined}
+    />
+  );
+
+  return fullPage ? (
+    <>
+      <EuiPage paddingSize="m">
+        <HeaderControl
+          controls={[
+            {
+              description: i18n.translate('workspace.list.description', {
+                defaultMessage:
+                  'Organize collaborative projects with use-case-specific workspaces.',
```

Comment:
```suggestion
                  'Organize collaborative projects with specialized workspaces.',
```

Reply:
this is comes from UI, so i would like keep it as it is